test/extensions/quic_listeners/quiche/platform/BUILD

```diff
@@ -137,3 +140,23 @@ envoy_cc_test_library(
         "@com_googlesource_quiche//:quic_platform_base",
     ],
 )
+
+envoy_cc_test_library(
+    name = "quic_platform_epoll_impl_lib",
+    hdrs = ["quic_epoll_impl.h"],
+    deps = ["@com_googlesource_quiche//:epoll_server_lib"],
+)
+
+envoy_cc_test_library(
+    name = "quic_platform_epoll_clock_lib",
+    srcs = [
```

Comment:
nit: Fold "srcs" and "hdrs" to single line?

Reply:
done